Player/player.tscn

```diff
@@ -1,10 +1,19 @@
-[gd_scene load_steps=4 format=3 uid="uid://cwibpj61yuhlv"]
+[gd_scene load_steps=7 format=3 uid="uid://cwibpj61yuhlv"]
 
 [ext_resource type="Script" path="res://Player/player.gd" id="1_62xea"]
 [ext_resource type="Texture2D" uid="uid://cw7qc4s44biy6" path="res://Player/character_robot_walk0.png" id="2_uka3s"]
 
 [sub_resource type="RectangleShape2D" id="RectangleShape2D_milye"]
-size = Vector2(43, 69.75)
+size = Vector2(43, 48.875)
+
+[sub_resource type="RectangleShape2D" id="RectangleShape2D_1arvl"]
```

Comment:
I believe we'll need the collisionshape of the hurtbox to extend beyond the collisionshape of the character or it won't detect any enemies.